package.json

```diff
@@ -16,13 +16,13 @@
     "yarn": ">=1.21.1"
   },
   "scripts": {
-    "clean": "npx rimraf .awcache .cache-loader build node_modules/.cache transpiled",
-    "build": "NODE_OPTIONS=--max_old_space_size=8192 webpack --bail --mode=production",
+    "clean": "npx rimraf .awcache .cache-loader build node_modules/.cache transpiled && cd app/scripts/modules/app/ && npm run clean",
+    "installDeps": "yarn --cwd app/scripts/modules/app",
+    "build": "npm run installDeps && NODE_OPTIONS=--max_old_space_size=8192 webpack --bail --mode=production --config app/scripts/modules/app/webpack.config.js",
```

Comment:
wait, where do the webpack bundles end up?

Reply:
They end up in `app/scripts/modules/app/`. I'm going to fix up `clean` after completing the migration.

Reply:
this will be a problem for folks using Docker containers.  We should either move, or symlink the output to `/build/webpack`, or update any builds and tooling that expects the output to be off the root directory

Reply:
Fixing it now.